etna/packages/etna-js/spec/setup.js

```diff
@@ -1,10 +1,31 @@
 import 'regenerator-runtime/runtime';
 // suppresses a specific React warning (comment raf out to bring it back)
+import nock from 'nock';
+import { stringifyRequest } from 'nock/lib/common';
 const raf = (global.requestAnimationFrame = (cb) => setTimeout(cb, 0));
 
 const Enzyme = require('enzyme');
 const EnzymeAdapter = require('enzyme-adapter-react-16');
 // Setup enzyme's react adapter
 Enzyme.configure({ adapter: new EnzymeAdapter() });
 
-global.fetch = require('node-fetch');
\ No newline at end of file
+global.fetch = require('node-fetch');
+
+nock.disableNetConnect();
+nock.emitter.on('no match', (...args) => {
+  // nock has inconsistent parameters for this event based on how the match failed, which results in us having complex
+  // unpacking logic.
+  let [ _, options, bodyString ] = args;
+  if (options == null) {
+    ({ options } = args[0]);
+    if (options == null) {
+      options = args[0];
+    }
+  }
+
+  const reqString = stringifyRequest(options, bodyString);
+  console.error('got request with no match', reqString);
+})
+
+// nodejs equivalent
+global.FormData = URLSearchParams;
```

Comment:
This fixes another issue I noticed where we were unable to match form data requests correctly.  It seems that in nodejs the correct replacement is URLSearcParams